src/openms/thirdparty/IsoSpec/IsoSpec/platform.h

```diff
@@ -0,0 +1,81 @@
+
+#pragma once
+
+#if !defined(ISOSPEC_BUILDING_R)
+#define ISOSPEC_BUILDING_R false
+#endif
+
+#if !defined(ISOSPEC_BUILDING_CPP)
+#define ISOSPEC_BUILDING_CPP true
+#endif
+
+#if !defined(ISOSPEC_BUILDING_PYTHON)
+#define ISOSPEC_BUILDING_PYTHON false
+#endif
+
+#if !defined(ISOSPEC_BUILDING_OPENMS)
+#define ISOSPEC_BUILDING_OPENMS false
+#endif
+
+#if defined(__unix__) || defined(__unix) || \
```

Comment:
maybe this should be wrapped in a 
```suggestion
#ifndef ISOSPEC_TEST_GOT_MMAN
#if defined(__unix__) || defined(__unix) || \
```